src/checkpoint/container/view_serialize.h

```diff
@@ -224,6 +236,16 @@ inline void serialize(
 
   DEBUG_PRINT_CHECKPOINT(s, "label=%s: size=%zu\n", label.c_str(), view.size());
 
+  // Kokkos::deep_copy between DynamicView instances is not yet implemented
+#if 0
```

Comment:
Do you need to keep this block around since it is explicitly disabled?

Reply:
Roughly, it's the implementation that I want to provide when the underlying bits are ready